endpoints/employees.md

```diff
@@ -120,3 +123,262 @@ curl --location --request GET "https://api.gearbox.com.au/public/v1/employees" \
 --header 'Accept: application/json' \
 --header 'Authorization: Bearer $ACCESS_TOKEN'
 ```
+
+## Create employees
+
+`POST /public/v1/employees` creates a new employee with the provided information.
+
+Please note:
+
+- Content-Type header `Content-Type: application/json` is required when POSTing employees into Gearbox.
+- Snake case object keys are required.
+
+### Request
+
+```
+URL: https://api.gearbox.com.au/public/v1/employees
+Method: POST
+Content-Type: application/json
+Authorization: Bearer $ACCESS_TOKEN
+
+{
```

Comment:
I believe the following fields are missing from this example:

- notes
- terminated
- next_of_kin
- next_of_kin_number